lib/std/fs/test.zig

```diff
@@ -1969,6 +1994,31 @@ test "delete a setAsCwd directory on Windows" {
     tmp.parent_dir.close();
 }
 
+test "use Lock.none on Windows" {
+    if (native_os != .windows) return error.SkipZigTest;
```

Comment:
This test can be deleted since it is now redundant with the `"use Lock.none to unlock files"` test

Reply:
oh whoops, I thought I did. I'll get on that.